web/src/components/PoisMobile.tsx

```diff
@@ -158,24 +188,14 @@ const PoisMobile = ({
             <StyledIcon src={ArrowBackspaceIcon} directionDependent />
           </BackNavigation>
         )}
-        {/* To use geolocation in a development build you have to start the dev server with "yarn start --https" */}
-        <GeolocateControl
-          style={{
-            bottom: bottomActionSheetHeight,
-            position: 'absolute',
-            right: 0,
-          }}
-          positionOptions={{ enableHighAccuracy: true }}
-          trackUserLocation
-          position='bottom-right'
-        />
       </MapView>
       <BottomActionSheet
         title={!currentFeatureOnMap ? t('listTitle') : undefined}
         toolbar={toolbar}
         ref={sheetRef}
         setBottomActionSheetHeight={setBottomActionSheetHeight}
-        direction={direction}>
+        direction={direction}
+        sibling={<GeoLocateContainer id='geolocate' direction={direction} ref={geolocatePosition} height={height} />}>
```

Comment:
Can we just use one spelling? I found at least `GeoLocate`, `geolocate` and `geolocator`. I think we should stick to the one used by the library, so `Geolocate`/`geolocate`.